modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -0,0 +1,353 @@
+* Copies the original content of the configuration files from the _current_ and _new_ Mule distributions to the following folders so you can compare them and identify the changes:
++
+`${MULE_HOME}/upgrade-tool/mule-configs-files/ORIGINAL_CONFIG_FILES_OLD_DISTRO/`
++
+`${MULE_HOME}/upgrade-tool/mule-configs-files/ORIGINAL_CONFIG_FILES_NEW_DISTRO/`
+
+If the Mule upgrade tool fails the upgrade process due to conflicts with the configuration files, follow the next steps to resolve the conflicts and continue the upgrade process.
+
+[[steps-to-solve-conflicts]]
+=== Steps to Solve Conflicts with Mule Configuration Files
+
+Follow these steps to resolve conflicts during an upgrade:
+
+. Inside the `${MULE_HOME}/upgrade-tool/mule-configs-files/conf/` folder of the current Mule distribution, locate one of the files that has a conflict.
+. Use any file comparison or `diff` tool to compare the contents of the files in the following paths:
+** `${MULE_HOME}/upgrade-tool/mule-configs-files/ORIGINAL_CONFIG_FILES_OLD_DISTRO/fileWithConflicts.conf`
+** `${MULE_HOME}/upgrade-tool/mule-configs-files/ORIGINAL_CONFIG_FILES_NEW_DISTRO/fileWithConflicts.conf`
+. After reviewing the differences, you can either accept or discard the changes:
+* If you decide to accept the changes, merge the content of the file `${MULE_HOME}/upgrade-tool/mule-configs-files/conf/fileWithConflicts_TO_REVIEW.conf` with the changes in the `${MULE_HOME}/upgrade-tool/mule-configs-files/ORIGINAL_CONFIG_FILES_NEW_DISTRO/fileWithConflicts.conf` file.
+* If you decide to discard the changes, skip this step.
++
+Note that discarding the changes might lead to using deprecated settings in the updated Mule distribution.
+. Rename the `${MULE_HOME}/upgrade-tool/mule-configs-files/conf/fileWithConflicts_TO_REVIEW.conf` file by removing the `_TO_REVIEW` suffix.
+. Repeat steps 1 to 4 for each configuration file that has the `_TO_REVIEW` suffix.
+. Retry running the `upgrade` subcommand.
++
+When the upgrade finishes successfully, the tool deletes the `${MULE_HOME}/upgrade-tool/mule-config-files` folder to have a clean environment for future upgrades.
+
+=== Unattended Upgrades
+
+You can perform an unattended upgrade by resolving the conflicts in a different environment before running the upgrade in the target environment. This procedure works only if the _current_ and _new_ Mule distributions are in the same version in both environments:
+
+. In an environment where user interaction is possible, review and resolve all the conflicts as explained in <<steps-to-solve-conflicts>>.
+. In the target environment, copy the files from `${MULE_HOME_WITH_USER_INTERACTION}/upgrade-tool/mule-configs-files/conf/` to `${MULE_HOME_WITHOUT_USER_INTERACTION}/upgrade-tool/mule-configs-files/conf/`.
+. Run the `upgrade` subcommand in the target environment.
+
+== Configure Logging for the Mule Upgrade Tool
```

Comment:
IMO, this section should come before the troubleshooting section i.e. "Troubleshooting File Conflicts".